one-pager-maker/src/entity/documentType.ts

```diff
@@ -18,11 +18,11 @@ export const documentSchema = z.object({
   reviewers: z.array(z.string()),
   url_privilege: privilegeSchema,
   deleted_at: z.instanceof(Timestamp).nullable(),
-  published_at: z.instanceof(Timestamp).nullable(),
+  published_at: z.instanceof(Timestamp).nullable().optional(),
   updated_at: z.instanceof(Timestamp),
   created_at: z.instanceof(Timestamp),
-  path: z.string(),
-  filename: z.string()
+  path: z.string().optional(),
+  filename: z.string().optional()
```

Comment:
```filename: z.string().optional() = filename?: z.string()?```

I don't know the typescript standard, but I personally prefer the shorter style. Please let me know if you have any preferences.

Reply:
It comes with the tsc error. I also prefer to write with shorter ver, but I doesn't pass the build test with short ver

Reply:
I tried it and it certainly is.